content/doc/book/managing/spawning-processes.adoc

```diff
@@ -0,0 +1,157 @@
+---
+layout: section
+---
+ifdef::backend-html5[]
+:notitle:
+:description:
+:author:
+:email: jenkinsci-docs@googlegroups.com
+:sectanchors:
+:toc:
+ifdef::env-github[:imagesdir: ../resources]
+ifndef::env-github[:imagesdir: ../../resources]
+:hide-uri-scheme:
+endif::[]
+
+= Spawning Processes From Build
+
+Sometimes you'd like to spawn a process from a build that lives longer than the build itself. For example, maybe a part of the build is to launch a new application server with the result of the build. When you do this, you often experience a problem where the build doesn't terminate; you'll see that shell script/ant/maven terminates as expected, but Jenkins just insists on waiting, as if it didn't notice that the build is over.
+
+Starting Jenkins 1.136, Jenkins detects this situation and instead of causing infinite block, it will just print out a warning and let you get going. But you should still understand what's causing this.
+
+ 
+== Why?
+
+The reason this problem happens is because of file descriptor leak and how they are inherited from one process to another. Jenkins and the child process are connected by three pipes (stdin/stdout/stderr.) This allows Jenkins to capture the output from the child process. Since the child process may write a lot of data to the pipe and quit immediately after that, Jenkins needs to make sure that it drained the pipes before it considers the build to be over. Jenkins does this by waiting for EOF.
```

Comment:
```suggestion
This happens because of file descriptor leak and how they are inherited from one process to another.
Jenkins and the child process are connected by three pipes ('stdin', 'stdout`, and `stderr`.)
This allows Jenkins to capture the output from the child process.
Since the child process may write a lot of data to the pipe and quit immediately after that, Jenkins waits for end-of-file (EOF) to be sure that it has drained the pipes before it terminates the build. 
```